zipline/algorithm.py

```diff
@@ -383,6 +384,40 @@ def __init__(self, *args, **kwargs):
 
         self.benchmark_sid = kwargs.pop('benchmark_sid', None)
 
+        self.capital_changes = kwargs.pop('capital_changes', {})
+        if self.capital_changes:
+            self.capital_changes = \
```

Comment:
Doesn't really matter for me. @abhijeetkalyan suggested they be passed in this format